cmd/lensserver/README.md

```diff
@@ -0,0 +1,9 @@
+# Lensserver
+
+This binary provides additional lenses for Prows spyglass log viewer.
```

Comment:
nit: "lens" is already Prow jargon, I think a link to https://github.com/kubernetes/test-infra/blob/master/prow/spyglass/write-a-lens.md and/or https://github.com/kubernetes/test-infra/blob/master/prow/spyglass/architecture.md#spyglass-lenses would be useful here.